examples/pony/multi_worker_forwarding/multi_worker_forwarding.pony

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Similar "why is this its own app?" question here.  What makes this different from the other two passthrough apps, and can they be consolidated?